frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -279,7 +282,13 @@ export const ProvidedAPIsPage = connect(inFlightStateToProps)((props: ProvidedAP
   const createProps =
     owned.length > 1
       ? {
-          items: owned.reduce((acc, crd) => ({ ...acc, [crd.name]: crd.displayName }), {}),
+          items: owned.reduce(
+            (acc, crd) =>
+              !isInternalObject(internalObjects, crd.name)
+                ? { ...acc, [crd.name]: crd.displayName }
+                : { ...acc },
```

Comment:
No reason to make a copy here

```suggestion
                : acc,
```